langfuse-3.67.yaml

```
@@ -0,0 +1,316 @@
+          DATABASE_URL: "postgresql://postgres:postgres@localhost:5432/test"
+          DIRECT_URL: "postgresql://postgres:postgres@localhost:5432/test"
+          NODE_ENV: "production"
+          PORT: "3030"
+          REDIS_HOST: "127.0.0.1"
+          REDIS_PORT: "6379"
+          REDIS_URL: "redis://127.0.0.1:6379"
+          REDIS_AUTH: ""
+          REDIS_CONNECTION_STRING: "redis://127.0.0.1:6379"
+          LANGFUSE_S3_EVENT_UPLOAD_BUCKET: "test-bucket"
+          CLICKHOUSE_URL: "http://localhost:8123"
+          CLICKHOUSE_USER: "testuser"
+          CLICKHOUSE_PASSWORD: "testpass"
+          NEXT_PUBLIC_LANGFUSE_CLOUD_REGION: ""
+          LANGFUSE_WORKER_HOST: "0.0.0.0"
+          LANGFUSE_WORKER_PORT: "3030"
+      pipeline:
+        - name: "Start Redis & Postgres, apply Prisma schema"
+          working-directory: /app
+          runs: |
+            redis-server --daemonize yes > /dev/null 2>&1
+
+            mkdir -p /var/lib/postgresql/data
+            chown build:build /var/lib/postgresql/data
+
+            mkdir -p /var/run/postgresql
+            chown build:build /var/run/postgresql
+
+            su build -c "initdb -D /var/lib/postgresql/data" > /dev/null 2>&1
+            su build -c "pg_ctl -D /var/lib/postgresql/data -o '-k /var/run/postgresql' start" > /dev/null 2>&1
+
+            su build -c "/usr/bin/psql \
+              -h /var/run/postgresql \
+              -d postgres \
+              -v ON_ERROR_STOP=1 <<-EOSQL
+                CREATE ROLE postgres WITH LOGIN SUPERUSER PASSWORD 'postgres';
+                CREATE DATABASE test OWNER postgres;
+            EOSQL" > /dev/null 2>&1
+
+            SCHEMA_FILE=$(find . -name "schema.prisma" -type f | head -1)
+            if [ -n "$SCHEMA_FILE" ]; then
+              echo "Found schema at: $SCHEMA_FILE"
+              npx prisma generate --schema="$SCHEMA_FILE" > /dev/null 2>&1 || true
+              npx prisma db push --accept-data-loss --skip-generate --schema="$SCHEMA_FILE" > /dev/null 2>&1
+            fi
+        - name: "Test worker startup via entrypoint"
+          uses: test/daemon-check-output
+          working-directory: /app
+          with:
+            setup: |
+              # Verify the worker files exist
+              if [ ! -f /app/worker/entrypoint.sh ]; then
+                echo "ERROR: /app/worker/entrypoint.sh not found"
+                ls -la /app/worker/ || echo "worker directory not found"
+                exit 1
+              fi
+              if [ ! -f /app/worker/dist/index.js ]; then
+                echo "ERROR: /app/worker/dist/index.js not found"
+                exit 1
+              fi
+            start: |
+              dumb-init -- ./worker/entrypoint.sh node worker/dist/index.js
+            timeout: 60
+            expected_output: |
+              Listening: http://
+              Finished upserting default model prices
+
+# this package updates and introduces CVE's very frequently
+update:
+  enabled: false
```

Needs exclude-reason:

```suggestion
  enabled: false
  exclude-reason:
```